Userland/Libraries/LibPDF/Filter.cpp

```diff
@@ -305,17 +306,15 @@ PDFErrorOr<ByteBuffer> Filter::decode_ccitt(ReadonlyBytes, RefPtr<DictObject> de
             damaged_rows_before_error = decode_parms->get_value(CommonNames::DamagedRowsBeforeError).get<int>();
     }
 
-    // FIXME: Do something with these.
-    (void)require_end_of_line;
-    (void)encoded_byte_align;
-    (void)columns;
-    (void)rows;
+    // FIXME: This parameter seems crucial when reading its description,
+    //        but I can't see any issue when ignoring it.
```

Comment:
Given we still have

```
Internal error while processing PDF file: CCITTDecoder: Corrupted stream, in 9 files, on 30 pages, 31 times
```

I'm not sure "can't see" is correct. (Maybe it's not due to this, but we don't know.) Maybe "Figure out what to to do with it." instead?

Reply:
Oh I didn't see these :thinking: